activerecord/lib/active_record/migration.rb

```diff
@@ -556,10 +556,12 @@ class Current < Migration #:nodoc:
     def self.inherited(subclass) #:nodoc:
       super
       if subclass.superclass == Migration
+        major = ActiveRecord::VERSION::MAJOR
+        minor = ActiveRecord::VERSION::MINOR
         raise StandardError, "Directly inheriting from ActiveRecord::Migration is not supported. " \
-          "Please specify the Rails release the migration was written for:\n" \
+          "Please specify the Active Record release the migration was written for:\n" \
           "\n" \
-          "  class #{subclass} < ActiveRecord::Migration[4.2]"
+          "  class #{subclass} < ActiveRecord::Migration[#{major}.#{minor}]"
```

Comment:
I think we need to keep 4.2 here. That was the only version that didn't have the version subclasses and migrations that still are using direct inheritance were probably created in 4.2.

But I'm not sure, new migrations being written right now would also see the same exception and in that case `#{major}.#{minor}` makes sense.

It would be good to have a way to differentiate, but I don't think we have and there is more harm in using 6.1 migration to code that was written in 4.2 than using 4.2 for code just being written.

What do you think?